grace/evaluation/process.py

```diff
@@ -215,11 +183,11 @@ def assume_annotations_from_dummy_predictions(G: nx.Graph) -> None:
     """Translates between dummy predictions & GT labels."""
 
     for _, node in G.nodes(data=True):
-        pred_label = node[GraphAttrs.NODE_PREDICTION][Prediction.LABEL]
+        pred_label = node[GraphAttrs.NODE_PREDICTION].label()
```

Comment:
No need to call this
```suggestion
        pred_label = node[GraphAttrs.NODE_PREDICTION].label
```

Reply:
Sorry, my oversight!